src/main/webapp/queue.js

```diff
@@ -159,7 +159,35 @@ async function getVideoData(id) {
         console.log(video);
       });
   getRoomQueue(roomParameters);
-  document.getElementById('linkArea').value = '';
+}
+
+// Retrieves search data from servlet and formats results in HTML
+async function getSearchQuery(keywords) {
+  if (keywords == '') {
+    console.log('NO KEYWORDS');
+    return;
+  }
+  await fetch('/keywordSearch?words=' + keywords)
```

Comment:
can it be a list of keywords?

Reply:
It should be a string because of how the program takes input from an input area. I've included some type checking to make sure I get a string instead of a list.